scripts/verify_env.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""
+Used to check missing deployed lambda env variables with local environment variables.
+Does not check the values of the variables, limited to a check if they key is present.
+
+"""
+import os
+import boto3
+import json
+
+
+lambda_client = boto3.client("lambda")
+
+
+def get_local_lambda_environment_keys():
+    env = ["ADMIN_USER_EMAILS", "DSS_ES_ENDPOINT", "GOOGLE_APPLICATION_CREDENTIALS", "GOOGLE_APPLICATION_SECRETS"]
+    for name in os.environ['EXPORT_ENV_VARS_TO_LAMBDA'].split():
+            env.append(name)
+    return env
+
+
+def get_ssm_lambda_environment(stage):
+    parms = lambda_client.get_function_configuration(
+        FunctionName=f"dss-{stage}"
+    )['Environment']['Variables']
+    return parms
+
+
+def compare_local_stage(stage=None):
```

Comment:
If stage is anything other than `None`, this could run into a conflict with the currently sourced environment.  Either delete it as an input arg and have this script only ever be used for the current stage, or source the env for the stage it's checking.